tasks/main.yml

```diff
@@ -9,7 +9,7 @@
     body: 'filter[host]={{ inventory_hostname }}'
     status_code: [200]
     timeout: '{{ new_relic_api_timeout | int }}'
-    validate_certs: no  # Add this line to skip SSL certificate validation
+    validate_certs: '{{ ssl_skip }}'   # Add this line to skip SSL certificate validation
```

Comment:
That's not a particular good name. If I set `ssl_skip` to `yes`, will it skip the SSL check?  

Why not just call it `validate_certs`?